File: ticket/src/app/feature/ticket/ticket.ts
import { Tip } from "../tip/tip";

export interface Ticket {
  ticketId: number,
  modulo: string,
  motivo: string,
  descripcion: string,
  userId: number,
  telefono: string,
  created: Date,
  enabled: boolean,
  tips:Tip[],
  // nameUser:string
}

File: ticket-b/src/main/java/yavirac/ticket/feature/ticket/Ticket.java
package yavirac.ticket.feature.ticket;

import java.sql.Timestamp;
import java.util.HashSet;
import java.util.Set;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.MappedCollection;
import org.springframework.data.relational.core.mapping.Table;

import lombok.Data;

@Data
@Table("ticketss")
public class Ticket {

    @Id
    private long ticketId;
    private Long userId;
    private String modulo;
    private String motivo;
    private String descripcion;
    private String telefono;
    private Timestamp created;
    private boolean enabled;

    @MappedCollection(idColumn = "ticket_id")
    private Set<TicketTip> tips = new HashSet<>();

}